content/docs/replenishment/purchase orders/index.md

```diff
@@ -0,0 +1,537 @@
++++
+title = "Purchase Orders"
+description = "Requesting stock from external suppliers."
+date = 2025-09-26
+updated = 2025-09-26
+draft = false
+weight = 44
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Intro
+
+## Viewing Purchase Orders
+
+### Go to the Purchase Orders Menu
+
+First you need to have purchase orders enabled for your store.
+
+1. Go to the `Manage` menu in the navigation panel
```

Comment:
Can you link to the store prefs page pls: `/docs/manage/facilities/#store-preferences`